releases/release-1.18/README.md

```diff
@@ -0,0 +1,71 @@
+# Kubernetes 1.18
+
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.18/README.md)
+* [Release Team](https://git.k8s.io/sig-release/releases/release-1.18/release_team.md)
+* [Meeting Minutes]() (join [kubernetes-sig-release@] to receive meeting invites)
+* [v1.18 Release Calendar][k8s118-calendar]
+* Contact: [#sig-release] on slack, [kubernetes-release-team@] on e-mail
+* [Internal Contact Info]() (accessible only to members of [kubernetes-release-team@])
+
+#### Tracking docs
+
+* [Enhancements Tracking Sheet]()
+* [Bug Triage Tracking Sheet]()
+* [CI Signal Report]()
+* [Retrospective Document]()
+* [kubernetes/sig-release v1.18 milestone][]
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
+
+## tl;dr
+
+The 1.18 release cycle is proposed as follows:
+
+- **Monday, January 06**: Week 1 - Release cycle begins
+- **Tuesday, January 28**: Week 4 - [Enhancements Freeze]
+- **Thursday, March 05**: Week 9 - [Code Freeze]
+- **Monday, March 16**: Week 11 - Docs must be completed and reviewed
+- **Tuesday, March 24**: Week 12 - Kubernetes v1.18.0 released
+
+
+## Timeline
+
+
+| **What** | **Who** | **Jan** | **Feb** | **Mar** | **WEEK** | **CI SIGNAL** |
+| --- | --- | --- | --- | --- | --- | --- |
+| Start of Release Cycle | Lead | Mon 6 | | | week 1 | [master-blocking] |
+| Start Enhancements Tracking | Enhancements Lead | Tue 7 | | | | |
+| Schedule finalized | Lead | Fri 10 | | | | |
+| Team finalized | Lead | Fri 17 | | | week 2 | |
+| 1.18.0-alpha.1 released | Branch Manager | Tue 21 | | | | |
+| Start Release Notes Draft | Release Notes Lead | Tue 21 | | | week 3 | |
+| **Begin [Enhancements Freeze]** (EOD PST) | Enhancements Lead | Tue 28 | | | week 4 | [master-blocking], [master-informing] |
+| 1.18.0-alpha.2 released | Branch Manager | | Tue 4 | | week 5 | |
+| 1.18.0-alpha.3 released | Branch Manager | | Tue 11 | | week 6 | |
+| release-1.14 jobs removed | Branch Manager | | Tue 18 | | week 7 | |
```

Comment:
the 1.14 job removed will be on another date from the 1.18 job creation? in the past, it was on the same day. any reason?

Reply:
oh, now it is in the same day :) @alejandrox1 disregard my comment above

Reply:
haha sorry about that. But thank you for double checking me!